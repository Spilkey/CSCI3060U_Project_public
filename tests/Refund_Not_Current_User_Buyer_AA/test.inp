login
user01
refund
user032342
UserCreditMax
254.00
logout
exit