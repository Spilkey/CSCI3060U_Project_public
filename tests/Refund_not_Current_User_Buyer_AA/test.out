00_user01__________AA_000000.00
00_user04__________SS_000000.00